src/Services/Air/Air.js

```diff
@@ -39,7 +39,74 @@ module.exports = (settings) => {
     },
 
     importPNR(options) {
-      return service.importPNR(options);
+      return service.importPNR(options)
+        .catch((err) => {
+          // Checking for error type
+          if (
+              !(err instanceof AirRuntimeError) ||
+              (err.data.faultstring !== 'No provider reservation to import.')
+          ) {
+            return Promise.reject(err);
+          }
+          // Creating passive segment to import PNR
+          const terminal = createTerminalService(settings);
+          const segment = {
```

Comment:
Let's move this segment creation to utils as function `getPassiveSegmentCommand` or something like that.